cfme/configure/access_control/__init__.py

```diff
@@ -836,11 +836,13 @@ def create(self, description=None, role=None, tenant="My Company", ldap_credenti
                 not having appropriate permissions OR delete is not allowed
                 for currently selected user
         """
-        if self.appliance.version < "5.8":
-            flash_blocked_msg = ("Description has already been taken")
-        else:
-            flash_blocked_msg = "Description is not unique within region {}".format(
-                self.appliance.server.zone.region.number)
+        flash_blocked_msg = (
+            "Description is not unique within region {}".format(
+                self.appliance.server.zone.region.number
+            )
+            if description
```

Comment:
It's valid but we should not go more than three line for inline `if else` block. It's difficult to read...